src/_pytest/fixtures.py

```diff
@@ -251,18 +252,18 @@ def get_parametrized_fixture_keys(item: nodes.Item, scope: Scope) -> Iterator[_K
         # cs.indices.items() is random order of argnames.  Need to
```

Comment:
TODO: Comment is now misleading, as we don't use cs.indices anymore.

Reply:
Right, so fix the comment.

Reply:
Arr I definitely did not mean to write "Right, so fix the comment." - I meant "Right, need to fix the comment." (just to mark it)